src/Avalonia.Base/Threading/IDispatcher.cs

```diff
@@ -26,30 +26,5 @@ public interface IDispatcher
         /// <param name="action">The method.</param>
         /// <param name="priority">The priority with which to invoke the method.</param>
         void Post(Action action, DispatcherPriority priority = default);
-
-        /// <summary>
-        /// Posts an action that will be invoked on the dispatcher thread.
-        /// </summary>
-        /// <param name="action">The method.</param>
-        /// <param name="arg">The argument of method to call.</param>
-        /// <param name="priority">The priority with which to invoke the method.</param>
-        void Post(SendOrPostCallback action, object? arg, DispatcherPriority priority = default);
```

Comment:
a curiosity, why did you remove this method? It was added to avoid allocating clousure.

Reply:
We could re-add it by introducing another specialized DispatcherOperation class, I guess.